tests/python/unittest/test_ir_container.py

```diff
@@ -34,6 +34,21 @@ def test_array_save_load_json():
     assert a_loaded[1].value == 2
 
 
+def test_dir_array():
+    a = tvm.runtime.convert([1, 2, 3])
+    dir(a)
```

Comment:
It would be better to check its outputs instead of just letting it go. It could be at least `assert dir(a)`.